commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -33,6 +33,7 @@ trait UnicycleHardware {
 
   val forwardPosition: Signal[Length]
   val turnPosition: Signal[Angle]
+  val position: Signal[Point]
```

Comment:
That would be problematic, since my class returns a PeriodicSignal of position. This  means that to get the position as a regular signal (which we will need to for initial positions), we will have to convert the position to a polling signal, which would be very convenient to do here, rather than every place that we need to get position. It also makes logging easier.